config/prod/minidream-rstudio-2023-ec2-alb.yaml

```diff
@@ -8,8 +8,6 @@ stack_name: "minidream-rstudio-2023-ec2-alb"
 dependencies:
   - "prod/minidream-rstudio-2023-ec2.yaml"
 parameters:
-  Department: "SCCE"
```

Comment:
You need a new version of the aws-infra script for that.

Reply:
ok, then i'm just gonna remove the spacing in the tags